.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+# To boot the app run the following:
+# docker-compose run auto-gpt
+version: '3.9'
+
+services:
+  auto-gpt:
+    depends_on:
+      - redis
+    build:
+      dockerfile: ./Dockerfile
+      context: ../
+    image: auto-gpt
+    tty: true
+    env_file:
+      - ../.env
```

Comment:
I'm not sure it makes sense to load this file into the container environment, because it gets loaded at startup. If the user changes the `.env` from within the devcontainer, which they likely will do at some point, they would have to restart the container to let the new config take effect.
```suggestion
```

Reply:
Fair enough, I had it left in as an artifact from the release build